seshdash/tasks.py

```diff
@@ -109,6 +108,13 @@ def get_BOM_data():
             if v_client.IS_INITIALIZED:
                         bat_data = v_client.get_battery_stats(int(site.vrm_site_id))
                         sys_data = v_client.get_system_stats(int(site.vrm_site_id))
+
```

Comment:
this bit was a removed. Can you pull the master first?
logger.debug("got raw date %s with timezone %s"%( sys_data['VE.Bus state']['timestamp'], site.time_zone )) date = time_utils.epoch_to_datetime(float(sys_data['VE.Bus tz=site.time_zone) logger.debug("saving before localize BOM data point with time 
what it looks like